portieris.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: portieris
+  version: 0.13.19
+  epoch: 0
+  description: A Kubernetes Admission Controller for verifying image trust.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - go
```

Comment:
this can be removed go/build pipeline already includes this. (https://github.com/chainguard-dev/melange/blob/c63354ae90657f4b28e71aaaa9f4543273116a6b/pkg/build/pipelines/go/build.yaml#L3-L7)

Note: If we need `build-base` at build time and then we can only include that in here.